MekHQ/src/mekhq/campaign/Quartermaster.java

```diff
@@ -697,14 +787,18 @@ public void sellArmor(Armor armor, int points) {
 
         Money cost = armor.getActualValue().multipliedBy(saleProportion);
 
-        getCampaign().getFinances().credit(TransactionType.EQUIPMENT_SALE, getCampaign().getLocalDate(),
-                cost, "Sale of " + points + " " + armor.getName());
+        getCampaign().getFinances()
+              .credit(TransactionType.EQUIPMENT_SALE,
+                    getCampaign().getLocalDate(),
+                    cost,
+                    "Sale of " + points + " " + armor.getName());
```

Comment:
You really should not be reformatting the whole file while working on it. Keep reformats to only the code you are working on.